src/CreateEntityMappingView/CreateEntityMappingView.js

```diff
@@ -0,0 +1,258 @@
+        if(connectionIncomplete) {
+            return connectionIncomplete;
+        }
+        let attributeFields = "";
+        if (this.props.eventTypeAttributes && this.props.entityTypeAttributes) {
+            attributeFields = this.loadAttributes();
+        }
+        return (
+            <div>
+                <Header title={"Create Entity Mapping"}/>
+                <div className={AppStyles.elementMarginTop}>
+                    <Container>
+                        <Row>
+                            <Col md={6}>
+                                <SelectField
+                                    value={this.state.selectedEventTypeId}
+                                    onChange={
+                                        (event, index, selectedEventTypeId) =>
+                                            this.handleEventTypeChange(selectedEventTypeId)
+                                    }
+                                    floatingLabelText="Select Event Type"
+                                    fullWidth={true}>
+                                    {CreateEntityMappingView.getMenuItems(eventTypes)}
+                                </SelectField>
+                            </Col>
+                            <Col md={6}>
+                                <SelectField
+                                    value={this.state.selectedEntityTypeId}
+                                    onChange={
+                                        (event, index, selectedEntityTypeId) =>
+                                            this.handleEntityTypeChange(selectedEntityTypeId)
+                                    }
+                                    floatingLabelText="Select Entity Type"
+                                    fullWidth={true}>
+                                    {CreateEntityMappingView.getMenuItems(entityTypes)}
+                                </SelectField>
+                            </Col>
+                        </Row>
+                        {attributeFields}
+                        <div className={css(AppStyles.textAlignCenter)}>
+                            <RaisedButton
+                                label="Abort"
+                                icon={<IconCancel/>}
+                                className={css(AppStyles.marginAllSites)}
+                                secondary={true}
+                            />
+                            <RaisedButton
+                                label="Save"
+                                icon={<IconSave/>}
+                                onTouchTap={this.prepareMappingsForSending}
+                                className={css(AppStyles.marginAllSites)}
+                                primary={true}
+                            />
+                        </div>
+                    </Container>
+                </div>
+            </div>
+        );
+    }
+}
+
+export default connect.defaults({fetch: ConnectionComponent.switchFetch})(props => ({
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "props". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)